packages/sanity/src/core/bundles/util/util.test.ts

```diff
@@ -0,0 +1,43 @@
+// Assuming Jest is used for unit testing and getBundleSlug is properly mocked
```

Comment:
I'm a little confused by this comment. Does this mean that these tests still need mocking for `getBundleSlug`? Or is this just clarifying what's going on?

Reply:
hehe removed, asked chatgpt for the initial test structure and copy pasted, missed removing that piece. There is no need to mock getBundleSlug